beets/dbcore/query.py

```diff
@@ -447,6 +447,34 @@ def match(self, item):
         return any([q.match(item) for q in self.subqueries])
 
 
+class NotQuery(Query):
+    """A query that matches the negation of its `subquery`, as a shorcut for
+    performing `not(subquery)` without using regular expressions."""
```

Comment:
Tiny style nit: PEP8 suggests putting the closing `"""` on its own line (for some reason).